core/src/test/java/org/infinispan/context/InvocationContextTest.java

```diff
@@ -96,9 +96,8 @@ public void run() {
       th.join();
       tm.resume(tx);
       tm.rollback();
-      assert throwables.size() == 1;
 
-      for (Throwable thr : throwables) thr.printStackTrace();
+      assert throwables.size() == 1;
```

Comment:
Can we use `assertEquals` instead? Gives information about received value if it fails.